draft-ietf-httpbis-priority.md

```diff
@@ -634,6 +634,32 @@ signal the intended priority to the client by including the Priority field in
 a PUSH_PROMISE or HEADERS frame.
 
 
+# Retransmission Scheduling
+
+A transport may detect packet losses and then retranmit lost information to
+provide reliability. Examples include TCP and QUIC. While this document
+specifies HTTP layer prioritization, its effectiveness can be further achieved
+if transport layer also respect such priorities. Besides scheduling new data,
+transport implementations MAY also consider stream priorities during
+retransmission. In this section we will use QUIC as an example to discusss a
+couple scenarios an implementation need to consider.
```

Comment:
```suggestion
Transport protocols such as TCP and QUIC provide reliability by detecting packet
losses and retransmitting lost information. While this document specifies
HTTP-layer prioritization, its effectiveness can be further enhanced if the
transport layer factors priority into scheduling both new data and
retransmission data. The remainder of this section illustrates this using QUIC
as an example.
```

This text you proposed is good. This suggestion is editorial to tighten things up a little. I've also removed the MAY since levying requirements on a transport protocol seems odd for this draft. IMO it should be enough to demonstrate the advantages and let people make up their own minds on what to do.